README.md

```diff
@@ -108,6 +108,13 @@ You can then login with sub `admin@example.com` and password `admin`.
 
 ### Run frontend
 
+#### Locally
+To run the frontend locally :
+
+`cd src/frontend/apps/drive && yarn && yarn build-theme && yarn dev`
```

Comment:
```suggestion
`cd src/frontend/ && yarn && cd apps/drive && yarn build-theme && yarn dev`
```

Reply:
But also think the main `package.json` file should have the `build-theme` command.